common/static/common/templates/discussion/thread.underscore

```diff
@@ -16,8 +16,19 @@
             <% } %>
             <div class="post-btn-count-wrapper">
                 <ul class="post-info-list">
-                    <li>By <strong>Babar Hameed</strong></li>
-                    <li>4 days ago</li>
+                    <li>By <strong>
+                    <% if (typeof username !== 'undefined') { %>
```

Comment:
Why are we checking for the type of username and not value? just curious

Reply:
if `username` is not defined then it will give an error, so we have to check its type first.